### ISSUE TITLE
PIXI renderer / textures not disposed after exiting room

### ISSUE THREAD

**Issue by EnumC** [64bitpandas/AtomBlast.io#11]
Critical uncaught error. PIXI renderer / textures not disposed after exiting room, resulting in unable to join a new game. PLAYER_SPRITES not unloaded. 

Potential fixes:
Destroy instance of stage + renderer upon exit. 

```
app.stage.destroy(true);
app.stage = null;

app.renderer.destroy( true );
app.renderer = null;
```

OR

Reset Loader
`loader.reset()`

![screen shot 2018-07-05 at 11 38 26 pm](https://user-images.githubusercontent.com/5799869/42363812-c8059570-80ad-11e8-8f78-e71e8d217b91.png)

**Comment by 64bitpandas** [comment]
We are already stopping the renderer on quit, but this does not unload the resources. Probably the easiest fix would be simply to check if the resource already exists before attempting to load it.

**Comment by EnumC** [comment]
@dbqeo It might be cleaner to dispose the renderer when PIXI.application stops as not disposing the renderer might cause issues when we reinitialize everything including the player. This problem seems to go deeper as the game would not start even when resources are not reloaded, and no errors are logged in the console. 
Update: Okay so it seems that app variable is being unnecessarily reinitialized and same with loader. We probably need to shuffle around a few things so we can start and stop the render without having to reinitialize everything.

**Comment by EnumC** [comment]
Fixed in commit 5e59af8 + 401318c by fixing what is being called when "Start Game" is pressed, so methods that only needs to be called once will not be reinvoked.

**Comment by 64bitpandas** [comment]
This issue was moved to BananiumLabs/AtomBlast.io#11
